### DISCUSSION TITLE
Why is my item upload progress stuck at 0%

### DISCUSSION THREAD

**Discussion by ybinstock** [mozilla/hubs#2715] [General Questions & Discussion] [answered]
I'm trying to upload an animated avatar glb file to my spoke scene, but it's getting stuck at 0% in the uploading file popup modal. What's strange is that if I create a room with the scene and upload the same model, it gets rendered in the scene without a problem. Does anyone have any idea what might be causing this issue?

Attached is the relevant glb file in a zipped folder (it doesn't allow me to attach a glb directly)
[glbZipped.zip](https://github.com/mozilla/hubs/files/4980711/glbZipped.zip)

**Comment by robertlong** [comment, marked as answer]
It looks like this is being caused by a circular dependency in one of the bones. The official Three GLTFLoader doesn't have this issue, but my method of loading dependencies has introduced this error. I've filed an issue here: https://github.com/mozilla/Spoke/issues/1007 and we can continue tracking it there.
